config/env/development.js

```diff
@@ -80,7 +80,8 @@ module.exports = {
      *   type: "manual",
      *   ip: "0.0.0.0"
      *   username: "Administrator",
-     *   password: "secret"
+     *   password: "secret",
+     *   plazaPort: 9090
```

Comment:
For unknown reason, setting "plazaPort" raise the following error : 
backend | error: column machine.plazaPort does not exist

Replace by "plazaport" to avoid that